src/types/index.ts

```diff
@@ -74,7 +74,12 @@ export type PreviewResponse = {
   rows: ResultRow[];
 };
 
-export type Response = LoadResponse | PreviewResponse;
+export type CheckNullAidsResponse = {
```

Comment:
I think I missed this when cleaning names. I'll probably rename to `FindNullsResponse` for consistency

Reply:
`CheckAidResponse` or `HasEmptyValuesResponse` sound nicer to me.

Reply:
so I wanted to have a "has..." name only on the level, where we're returning a boolean, so I'll do `CheckAidResponse` if that's preferred